.github/workflows/release.yml

```diff
@@ -11,6 +11,18 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Bazel caches
+        uses: actions/cache@v2
+        with:
+          # Only restore the repository cache. Don't restore the build cache
+          # as stamped bazel builds won't rebuild targets when the workplace
+          # status variables change.
```

Comment:
Is my assumption here correct? We always want to build a release build from scratch so that the artifacts will be properly stamped.